pathfilters/pathfilters.go

```diff
@@ -99,17 +103,30 @@ func SDKFilter(sdk string, allowed bool) pathutil.FilterFunc {
 
 			}
 		} else {
-			return false, fmt.Errorf("Not Xcode project nor workspace file: %s", pth)
+			return false, fmt.Errorf("not Xcode project nor workspace file: %s", pth)
 		}
 
+		sdkMap := map[string]bool{}
 		for _, projectFile := range projectFiles {
-			pbxprojPth := filepath.Join(projectFile, "project.pbxproj")
-			projectSDKs, err := xcodeproj.GetBuildConfigSDKs(pbxprojPth)
+			project, err := xcodeproj.Open(projectFile)
 			if err != nil {
 				return false, err
 			}
 
-			for _, projectSDK := range projectSDKs {
+			var buildConfigurations []xcodeproj.BuildConfiguration
```

Comment:
Are TargetAttributes also needed to be checked?

Reply:
The intention was to keep the original behaviour, other than that i did not find a project where `SDKROOT` is defined as target attribute